src/index.js

```diff
@@ -65,22 +65,47 @@ const poll = () => {
     })
 }
 
+const clearLog = () => {
+  const request = new Request('/clearLog', {
+    method: 'POST',
+  })
+
+  fetch(request)
+    .then(response => {
+      if (response.status === 200) {
+        return response.json()
```

Comment:
as it stands you're not returning a response, but if you add my suggestion to return the log then this can stay :)

Reply:
Will remove this 👍

Reply:
maybe just call `display([])` so we wipe the log on FE